ci/jobs/synthetic-data-ingestion-dev.yml.j2

```diff
@@ -0,0 +1,22 @@
+jobs:
+  - name: synthetic-data-ingestion-dev
+    plan:
+      - put: meta
```

Comment:
We don't need the meta resource/steps/resource_type in this repo yet so these lines plus line 194 of `ci/meta.yml` the additions to `ci/resource_types.yml` and the reference in `ci/resources` can all just be removed please.